src/templates/base/2019/base_chapter.html

```diff
@@ -86,7 +87,7 @@
 {% macro render_byline() %}
   <div class="byline">{{ self.written_by_before() }}
   {% for author in metadata.get('authors') %}
-    <a class="author" href="{{ url_for('contributors', year=year, lang=lang, _anchor=author) }}">{{ config.contributors[author].name if author in config.contributors else author }}</a>{{ self.comma() if not loop.last }}
+    <a class="author" href="#author-{{ author }}">{{ config.contributors[author].name if author in config.contributors else author }}</a>{{ self.comma() if not loop.last }}
```

Comment:
I think this should still link to the Contributors page.

Reply:
Yeah but the contributor page doesn’t have the bio info so it’s actually more useful to the reader to link to bottom of the page.

But agree it is odd that it will like to different place than reviewer or contributor.

Alternatively, we could try to expose the author bios on the contributors page somehow on a separate PR. A modal or something? Also been thinking it would be good to somehow link contributors/reviewers/translators to chapters anyway. Either that or the long standing search functionality I’ve kept open to look at, at some point...

WDYT?

Reply:
Have revert this so we can merge. Will raise a separate issue to discuss.